packages/app-builder-lib/src/winPackager.ts

```diff
@@ -366,7 +369,9 @@ export class WinPackager extends PlatformPackager<WindowsConfiguration> {
       if (this.shouldSignFile(file)) {
         const parentDir = path.dirname(file)
         if (parentDir !== packContext.appOutDir) {
-          return new CopyFileTransformer(file => this.sign(file))
+          return new CopyFileTransformer(async file => {
+            await this.sign(file)
```

Comment:
Is this change intentional?
`file => this.sign(file)` returns `Promise<boolean>`
`async file => { await this.sign(file) }` returns `Promise<void>` due to the brackets

Reply:
I was getting linting issues that this callback expected a return type of `Promise<void>`, which is why I made the change, but these do not seem important, I will remove this if needed.